src/main/resources/db/migration/V1_0__initial_tables.sql

```diff
@@ -0,0 +1,12 @@
+create table narmesteleder
```

Comment:
Du er litt streng med størrelsen på feltene her. Det er kanskje bedre å gi de litt mer plass, så slipper vi feil pga at noen plutselig har en aktørid som er 12 i stedet for 11 tegn osv.